src/pages/module-builder/Toolbar.tsx

```diff
@@ -96,7 +96,7 @@ const Toolbar = ({
                     icon={<IoTrash />}
                     onClick={() => {
                         dispatch({
-                            type: ModuleActionType.REMOVE_SLIDE,
+                            type: ModuleActionType.RESET_SLIDE,
```

Comment:
The previous behavior was correct actually, this line should be reverted. Sorry about that! You can leave the updated reset.

Reply:
Should I include another reset button to this? Or leave this to a future implementation